### ISSUE TITLE
Show manuscript metadata in manuscript view

### ISSUE THREAD

**Issue by zmbq** [Scripta-Qumranica-Electronica/ScrollEditor#184]
Metadata from manifest.

Bronson, is this metadata (period, composition type, manuscript type, language, etc...) available? It needs to be provided for just one edition, not in the edition list.

**Comment by zmbq** [comment]
@Bronson-Brown-deVost , the metadata endpoint is not working, it fails due to a missing table : edition_iaa_manifest. I'm using the latest sqe-db docker image. I guess you should upload a newer image.

**Comment by Bronson-Brown-deVost** [comment]
Yes sorry, that must be the problem.  I will try to do it tonight, I am working full speed right now on day job stuff, which has also been enjoyable.

Bronson
